apps/cowswap-frontend/src/modules/swap/containers/SafeTokenBanner.tsx

```diff
@@ -0,0 +1,43 @@
+import BRAIN_IMAGE from '@cowprotocol/assets/svg/brain.svg'
+import { useFeatureFlags } from '@cowprotocol/common-hooks'
+import { BannerOrientation, ClosableBanner, InlineBanner } from '@cowprotocol/ui'
+
+import { Settings } from 'react-feather'
+
+const SAFE_TOKEN = '0x5afe3855358e112b5647b952709e6165e1c1eeee'
+const SAFE_TOKEN_WARNING_STORAGE_KEY = 'safeTokenWarning:v0'
```

Comment:
Should we have this warning for the token on gnosis chain too?

Or, should we still have this warning now that the first day is passed?

Reply:
Good question :) 

I think we should remove. I will ask around

Reply:
done